langchain/src/agents/tools/aiplugin.ts

```diff
@@ -51,7 +51,7 @@ export class AIPluginTool extends Tool implements AIPluginToolParams {
 
     return new AIPluginTool({
       name: aiPluginJson.name_for_model,
-      description: `Call this tool to get the OpenAPI spec (and usage guide) for interacting with the ${aiPluginJson.name_for_human} API. You should only call this ONCE! What is the ${aiPluginJson.name_for_human} API useful for? ${aiPluginJson.description_for_human}`,
```

Comment:
From our testing it actually produces better results as it is currently

Reply:
Did you have better results with your changes?

Reply:
I think it's more reasonable to use xx_for_model in prompt rather than xx_for_human.
If we use xx_for_human in prompt, so what does xx_for_model mean for ?

Reply:
Notice they're used in the other properties, `name` and `apiSpec`. This description is meant to be a human-readable description of what this tool is for, not a description of how to use it (that's `apiSpec`)